scripts/updateVersion.ts

```diff
@@ -79,7 +79,7 @@ function updatePackageJsonVersion(cwd: string, newVersion: string) {
 function updateVersionByRegex(filePath: string, regex: RegExp, newVersion: string) {
     const oldFileContents = readFileSync(filePath).toString();
     const match = oldFileContents.match(regex);
-    if (!match) {
+    if (!match || !match[0] || !match[1]) {
         throw new Error(`Failed to find match for "${regex.source}".`);
     }
     const oldLine = match[0];
```

Comment:
We could use array destructuring here:

`const [oldLine, oldVersion] = match;`